afs/templates/views/components/reports/scenes/default.htm

```diff
@@ -3,8 +3,13 @@
 {% block report %}
 
 {% block body %}
-<div class="afs-report-section" data-bind="foreach: {data: data().sections, as: section, noChildContext: true}">
-    <h2 class="afs-report-section-title"><i data-bind="click: function() {toggleVisibility(visible[section.title])}" class="fa fa-angle-double-right toggle"></i> <span data-bind="text: section.title"></span></h2>
+<div class="afs-report-section" data-bind="foreach: {data: data().sections, as: 'section', noChildContext: true}">
```

Comment:
You'll want to do a pull here; I had a change to this sitting on my local branch that I had to push this morning, so. now there's a conflict.  Looks like you just expanded this out though, so it shouldn't be too difficult to resolve

Reply:
Merge conflict is resolved